src/test/java/uk/nhs/prm/e2etests/performance/RepoInPerformanceTest.java

```diff
@@ -102,59 +107,59 @@ private void assertMessagesAreInTransferCompleteQueue(int numberOfMessagesToBePr
     }
 
     private void sendMessagesToMq(List<RepoInPerfMessageWrapper> messagesToBeProcessed) {
-        var intervalBetweenMessagesSentToMq = getIntervalBetweenMessagesSentToMq();
+        int intervalBetweenMessagesSentToMq = getIntervalBetweenMessagesSentToMq();
         try {
-            var messageTemplate = ResourceUtility.readTestResourceFileFromEhrDirectory("small-ehr-4MB");
-            var counter = new AtomicInteger(0);
+            //
+            String messageTemplate = ResourceUtility.readTestResourceFileFromEhrDirectory("small-ehr-4MB");
+            AtomicInteger counter = new AtomicInteger(0);
             String smallEhr;
```

Comment:
Slitting hairs, I don't think there is a need for this variable. Having `inboundQueueFromMhs.sendMessage(getSmallMessageWithUniqueConversationIdAndMessageId(messageTemplate, conversationId), conversationId);` seems readable enough.

Additional to this, can't we significantly chop down this method name: `getSmallMessageWithUniqueConversationIdAndMessageId`.